### PR TITLE
SAK-45191: LESSONS: Export error NumberException when exporting IMS CC

### PR DESCRIPTION
**Pull request by jesusmmp**

### CODE REVIEW

**Review by adrianfish — APPROVED** [review]
Looks fine to me. Thanks.

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java [hunk 1]
```diff
@@ -141,7 +141,14 @@ private CCAssignmentItem getContents(String assignmentRef) {
         switch (typeOfGrade) {
             case SCORE_GRADE_TYPE:
                 Integer scaleFactor = assignment.getScaleFactor() != null ? assignment.getScaleFactor() : assignmentService.getScaleFactor();
-                double maxPoints = new Double(assignmentService.getMaxPointGradeDisplay(scaleFactor, assignment.getMaxGradePoint()));
+                String aux = assignmentService.getMaxPointGradeDisplay(scaleFactor, assignment.getMaxGradePoint());
+                String decSeparator = formattedText.getDecimalSeparator();
+                double maxPoints = 0.0;
+                try {
+                    maxPoints = Double.valueOf(StringUtils.replace(aux, decSeparator, ".")) * 10;
```
Comment (jesusmmp) [inline lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java:148]: maxPoints should be multiplied by 10. On 12.x and 19.x is on that way.

Comment (ottenhoff) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java:148]: this seems wrong. the scale factor doesn't have to be 10, right @ern?

Comment (jesusmmp) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java:148]: What should we do then? Should I remove * 10 and create another jira to investigate why sacleFactor is 100?

Comment (ern) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java:148]: There should be no reason to multiply this by 10 here

**Review by ern — CHANGES_REQUESTED** [review]

File: lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java (hunk 1 above)
Comment (jesusmmp) [reply lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/ccexport/AssignmentExport.java]: I have deleted * 10

**Review by ern — APPROVED** [review]

**Pull request merged**